src/label.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Label
+  attr_reader :id, :color, :title, :items
```

Comment:
Add attr_writer for the title and items according to the class diagram.
![image](https://user-images.githubusercontent.com/55840999/227062141-a8a00a93-e17f-4a27-a427-d5836646b252.png)